### PR TITLE
Architecture

### PR DESCRIPTION
**Pull request by ciaranjohnston**
As discussed in the architecture meeting on Friday, and again in the SIG Network Architecture meeting today, here is the PR for the architecture documentation in C4.

### CODE REVIEW

File: content/en/docs/architecture.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+title: "Nephio Architecture"
+type: docs
+weight: 5
+description: Reference for the Nephio Architecture
+---
+
+Some experiments on working with [C4 model](https://c4model.com/) to document Nephio.
+
+## Prerequisites
+To work with these PlantUML files, you will need [Graphviz](https://graphviz.org/download/) installed.
```
Comment (liamfallon) [inline content/en/docs/architecture.md:11]:
```suggestion
1. [Graphviz](https://graphviz.org/download/) is required to render some of the diagrams in this document.
```

File: static/images/architecture/level3-nephio-core-component.png
Comment (liamfallon) [inline static/images/architecture/level3-nephio-core-component.png:1]: The text on this diagram is very small and I had to download the image locally to read it.

File: content/en/docs/architecture.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Component Views
+
+### Nephio Core
+
+![Nephio Core Component View](/images/architecture/level3-nephio-core-component.png)
+
+Nephio core is a collection of operators and functions that perform the fundamental aspects of Nephio use cases, independent of the specifics of vendor implementations. 
+
+The controllers for OAI and Free5GC are represented here as while they are vendor extensions to Nephio they are for now part of the Nephio system.
```
Comment (liamfallon) [inline content/en/docs/architecture.md:34]:
```suggestion
The controllers for OAI and Free5GC are represented here. Although they are vendor extensions to Nephio, they are for now part of the Nephio system.
```

**Review by liamfallon — COMMENTED** [review]
Looks good.

**Review by adetalhouet — APPROVED** [review]
LGTM although last update is from me.

**Comment by liamfallon** [comment]
EasyCLA is blocking merge of PRs at the moment. A ticket has been raised with the LF: https://nephio.slack.com/archives/C03L6Q8LFHD/p1713270138006809

**Pull request merged**